ee/clickhouse/views/test/__snapshots__/test_clickhouse_experiment_secondary_results.ambr

```diff
@@ -1,4 +1,52 @@
 # name: ClickhouseTestExperimentSecondaryResults.test_basic_secondary_metric_results
+  '
+  /* user_id:46 celery:posthog.celery.sync_insight_caching_state */
```

Comment:
Do you know why are these snapshots being added here?

Reply:
Now that creating feature flags will create dashboards, the [post_save hook](https://github.com/PostHog/posthog/blob/master/posthog/models/insight_caching_state.py#L58) will be called for dashboard tiles